ner/src/main/java/edu/illinois/cs/cogcomp/ner/ClassifiersAndUtils/NfoldCrossvalidation.java

```diff
@@ -63,7 +68,7 @@ public void printNfoldCorrssvalidationNbAcc(int classesNum, double thres,
         System.out.println("\tConfusion matrix within confidence threshold:");
```

Comment:
be consistent -- either all STDOUT or all logger.  Please look at where this method is called from and decide whether usage indicates STDOUT or logger should be used.